src/LinSys/linsysmerger.ci

```diff
@@ -200,7 +200,7 @@ module linsysmerger {
       entry void wait4lhs() {
         when lhsbc_complete() serial "hyprelhs" { hyprelhs(); } };
       entry void wait4rhs() {
-        when rhsbc_complete() serial "hyprerhs" { hyprerhs(); } };
+        when rhsbc_complete_rhs() serial "hyprerhs" { hyprerhs(); } };
```

Comment:
Because we could find at least a single example that _does exactly the same change_ as this PR suggests and fails, that renders the change in this PR incorrect until we can convince ourselves to the contrary. Note that passing all regression tests is not a proof of correctness, since coverage is less than 100%.

You are not required at all to fix this problem. I would like you to try to reproduce it so I can be sure, that at least the problem can be reproduced elsewhere. I simply just wanted to alert you about the issue, that's all. If you have a better understanding of the issue, I welcome any insight, if not, that's totally fine too, but I don't think we should put in this change unless we can convince ourselves that it is correct.